test/factories/users.rb

```diff
@@ -25,5 +25,14 @@
 
       with_primary_membership
     end
+
+    trait :pek_admin do
+      after(:build) do |user|
+        SystemAttribute.create(id: 2, name: 'ertekeles_idoszak', value: 'NINCSERTEKELES')
```

Comment:
Do we need this here?

Reply:
I think we do. In application/_navbar.html.erb when the user is pek_admin, we run this code:
```ruby
<% unless SystemAttribute.offseason? %>
```
Which throws an error if SystemAttribute.season is not present. I'm not sure if this is a correct place to put that code, but it is necessary.

Reply:
This can be written like this
```
        create(:system_attribute_app_season)
```
though, it seems to be very extraneous to put here a system wide setting because it is used in a partial. Other option can be to put it to the test setup, which could be enough for now, but later when somebody takes a look at the test it can make them confusing, because it has nothing to do with the essence of it. The most reasonable for me to put it [here](https://github.com/kir-dev/pek-next/blob/master/spec/rails_helper.rb#L49L63) like with the semester.